packages/wrangler/templates/pages-template-worker.ts

```diff
@@ -126,19 +126,20 @@ export default {
           waitUntil: workerContext.waitUntil.bind(workerContext),
         };
 
-        const response = await handler(context);
+        const response: Response = await handler(context);
 
         // https://fetch.spec.whatwg.org/#null-body-status
-        return new Response(
-          [101, 204, 205, 304].includes(response.status) ? null : response.body,
-          { ...response, headers: new Headers(response.headers) }
-        );
+        return unlockResponse(response);
       } else if (__FALLBACK_SERVICE__) {
+        const response: Response = await env[__FALLBACK_SERVICE__].fetch(
```

Comment:
Similarly, the `fetch()` method here is already typed to return a `Promise<Response>` so no need for the explicit type.